README.md

```diff
@@ -212,6 +212,14 @@ It is possible to interrupt the watcher from another thread with:
 watcher.finish
 ```
 
+#### Get a proxy URL
+You can get a complete URL for connecting a kubernetes entity via the proxy.
+
+```ruby
+client.proxy_url('services', 'srvname', '5001-tcp', 'ns')
```

Comment:
I think port has to be only an int - don't think even port works & all proxies are tcp iirc.